chroma-agent/chroma-agent.spec

```diff
@@ -139,12 +133,17 @@ setenforce 0
 
 if [ $1 -eq 1 ]; then
     # new install; create default agent config
+    systemctl enable %{name}
     chroma-agent reset_agent_config
 elif [ $1 -eq 2 ]; then
-    systemctl enable %{name}
-    systemctl start %{name}
-    # upgrade; convert any older agent config
+    # upgrade; convert any older agent config and then restart
     chroma-agent convert_agent_config
+    systemctl enable %{name}
+    # it's tempting to restart the agent here, but this upgrade
+    # could have been initiated from the running chroma-agent and
+    # so restarting here would pull the rug out from under that
+    # upgrade procedure
+    #systemctl restart %{name}
```

Comment:
Do we need the commented out restart statement as well?

Reply:
Are you asking if we need it still commented out or if we need it uncommented?

Reply:
Is it worth keeping the ```#systemctl restart %{name}``` there v.s. just removing it.

Reply:
I left it there as an example of what we don't want to do